.devcontainer/Dockerfile

```diff
@@ -30,26 +33,40 @@ COPY .devcontainer/scripts/docker-client.sh /tmp/
 RUN /tmp/docker-client.sh $USERNAME
 
 # Install Docker
-RUN apt-get update && sudo apt-get install -y  apt-transport-https ca-certificates curl gnupg  lsb-release \
+RUN apt-get update && sudo apt-get install -y apt-transport-https ca-certificates curl gnupg lsb-release \
     && curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
     && echo "deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
     | tee /etc/apt/sources.list.d/docker.list > /dev/null \
-    && apt-get update && apt-get install -y docker-ce docker-ce-cli containerd.io
+    && apt-get update && apt-get install -y docker-ce docker-ce-cli containerd.io \
+    && apt-get clean -y && rm -rf /var/lib/apt/lists/*
```

Comment:
You clean on line 51 - wouldn't it be better to clean once at the end?

Reply:
No :-)
Each layer is independent, so if a run command doesn't end with the cleanup, the data is persistent in the layer. Cleaning in the end will be a separate layer that will only mask / mark as deleted but the container will still contain that data and its size will be larger.